### ISSUE TITLE
Output file folder 

### ISSUE THREAD

**Issue by i-sayuh** [NREL/LiAISON#12]
The output file folder where results are sent to does not exist. It is easy to create it yourself. However, to avoid any confusion with future users, I think this folder should already exist and not have to be manually created by the user.

**Comment by tjlca** [comment]
Good point. Will be added. It got deleted because Github does not support empty folders. @i-sayuh
